### PR TITLE
New package: scheme48-1.9.2

### PR DESCRIPTION
**Pull request by rc-05**

### CODE REVIEW

**Comment by not-chicken** [comment]
Hi @rc-05, thanks for your contribution!
 Seems like you accidentally pushed some extra commits. 
You can drop them using `git rebase -i HEAD~3` which will open up your text editor showing your last 3 commits. Replace the `pick` in front of commits with `drop`, then save and quit.  
Finally (force)push the changes here using `git push -f`.

File: srcpkgs/scheme48/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'scheme48'
+pkgname=scheme48
+version=1.9
+revision=2
```
Comment (Chocimier) [inline srcpkgs/scheme48/template]: `revision` is used to distinguish consecutive Void builds of program from same sources. Please set version to 1.9.2, revision to 1, drop `wrksrc` and adapt usages of $version.

File: srcpkgs/scheme48/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+archs="x86_64 x86_64-musl"
```
Comment (Chocimier) [inline srcpkgs/scheme48/template]: The problem is cross compiling, not tying to architecture. Use `nocross="failed to compile test program"` instead.

File: srcpkgs/scheme48/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+wrksrc="${pkgname}-${version}.${revision}"
+build_style=gnu-configure
+configure_args="--prefix=/usr"
+short_desc="Implementation of the R5RS Scheme standard"
+maintainer="rc-05 <rc23@email.it>"
+license="custom"
```
Comment (Chocimier) [inline srcpkgs/scheme48/template]: Please use `custom:scheme48`, as per Manual.md.

**Comment by rc-05** [comment]
Corrected some mistakes in the template file. Sorry for closing the pull request and re-opening it

**Comment by Chocimier** [comment]
Just small changes are needed before merge:
--prefix=/usr is set by build style by default, remove it.
File should end with newline character, but no empty line also.
Branch history should be rewritten so it contain one commit.

**Comment by rc-05** [comment]
> Just small changes are needed before merge:
> --prefix=/usr is set by build style by default, remove it.
> File should end with newline character, but no empty line also.
> Branch history should be rewritten so it contain one commit.

I'm having trouble squashing everything together as a single commit because of a merge commit that i did...

**Comment by Chocimier** [comment]
Following work just fine:

git checkout origin/master; git merge --squash rc-05/personal

How did you try?

**Comment by rc-05** [comment]
I've tried to rebase the commits by picking the most recent one and squashing everything together but it didn't work

**Pull request merged**